### ISSUE TITLE
VGG image annotator parser

### ISSUE THREAD

**Issue by lgvaz** [airctic/icevision#573]
## 🚀 Feature
Add a parser for the VGG Image Annotator format described [here](https://www.robots.ox.ac.uk/~vgg/software/via/)

**Suggestion:** Create the file `icevision/parsers/vgg_parser.py`. Use the other two implemented parsers as a structure guide.

The hardest part will be adding tests, for that we'll need to add some images and annotations under `samples/vgg` and follow similar tests to `tests/parsers/test_voc_parsers.py`

**Comment by jerbly** [comment]
Hey! I guess this one is for me! Happy to take it on. :)

**Comment by ai-fast-track** [comment]
Hey @jerbly ,

In IceVision [version 0.2.0](https://github.com/airctic/icevision/blob/master/CHANGELOG.md#changed-5), we unified `parsers.SizeMixin` functions `image_width `and `image_height `into a single function `image_width_height`. So, we need to make the following change in the `ViaParser` class:

We need to replace this:
```
    def height(self, o) -> int:
        return open_img(self.source/o[1]['filename']).shape[0]

    def width(self, o) -> int:
        return open_img(self.source/o[1]['filename']).shape[1]
```

with this:
```
    def image_width_height(self, o) -> Tuple[int, int]:
        return get_image_size(self.filepath(o))
```
We need to test the Parser against the latest IceVision version. Could you please double check on your end that everything works properly with the new API. 

Also I was wondering what is the most intuitive class name we should use. Either the `ViaParser` or `VIAParser`. If we follow both the COCO and the VOC Parsers convention, I would go with `VIAParser`

@lgvaz any thoughts on the naming?

**Comment by jerbly** [comment]
Hey, I saw that `image_width_height` change already. That won't be an issue - I had my project frozen on 0.1.4 while I was working on a bunch of other stuff. And, I'll use `VIAParser` for consistency. I'm starting to put this together now but I have a super-busy week IRL so it'll be a few days.

**Comment by jerbly** [comment]
@ai-fast-track , @lgvaz - I found the time to get this done after all, the PR is in... The 3.6 build is failing though and I don't think that's to do with me??

**Comment by lgvaz** [comment]
@jerbly The 3.6 build is failing though and I don't think that's to do with me??

Yeah, this is related to the latest numpy version which doesn't support 3.6 anymore

**Comment by lgvaz** [comment]
closed with #578
